Assets/Scripts/Authoring/PlayerMovementAuthoringComponent.cs

```diff
@@ -71,6 +71,29 @@ public class PlayerMovementAuthoringComponent : MonoBehaviour, IConvertGameObjec
         /// </summary>
         public float pushDecay = 0.0f;
 
+        /// <summary>
+        /// Duration of jumping grace period
+        ///     - Player can jump while not grounded, as long as they haven't been grounded longer than this amount of time
+        ///     - Make sure the value of jumpGraceTime < jumpCooldown. 
+        ///       Otherwise, double jumps are possible.
+        /// </summary>
+        public float jumpGraceTime = 0.0f;
+
+        /// <summary>
+        /// Time spent falling
+        /// </summary>
+        public float elapsedFallTime = 0.0f;
+
+        /// <summary>
+        /// Time spent falling
+        /// </summary>
+        public float jumpCooldown = 0.1f;
+
+        /// <summary>
+        /// Time elapsed since last jump. 
+        /// </summary>
+        public float timeElapsedSinceJump = 0.11f;
```

Comment:
Why does this value start greater than zero, is there an error or bug that occurred when it started at zero?

Reply:
Right, my bad! It's late and I was trying to reset the default of JumpCooldown